### PR TITLE
lora: init at v3.005

### PR DESCRIPTION
**Pull request by ofalvai**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [X] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/lo/lora/package.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lora";
+  version = "v3.005";
+
+  src = fetchFromGitHub {
+    owner = "cyrealtype";
+    rev = version;
+    repo = "lora";
```
Comment (SuperSandro2000) [inline pkgs/by-name/lo/lora/package.nix:15]:
```suggestion
    owner = "cyrealtype";
    repo = "lora";
    rev = version;
```

File: pkgs/by-name/lo/lora/package.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:8]:
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
- #315337
- https://nix.dev/guides/best-practices#recursive-attribute-set-rec

File: pkgs/by-name/lo/lora/package.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  pname = "lora";
+  version = "v3.005";
+
+  src = fetchFromGitHub {
+    owner = "cyrealtype";
+    repo = "lora";
+    rev = version;
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:15]:
```suggestion
    rev = finalAttrs.version;
```
- #315337
- https://nix.dev/guides/best-practices#recursive-attribute-set-rec

File: pkgs/by-name/lo/lora/package.nix [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  '';
+
+  meta = with lib; {
+    description = "Lora is a well-balanced contemporary serif with roots in calligraphy";
+    homepage = "https://github.com/cyrealtype/lora";
+    license = licenses.ofl;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ ofalvai ];
+  };
+}
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:40]:
```suggestion
})
```
- #315337
- https://nix.dev/guides/best-practices#recursive-attribute-set-rec

File: pkgs/by-name/lo/lora/package.nix [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    cp -R $src/fonts/ttf/*.ttf $out/share/fonts/truetype
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:33]:
```suggestion
  meta = {
```
- #208242
- #293767

File: pkgs/by-name/lo/lora/package.nix [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    description = "Lora is a well-balanced contemporary serif with roots in calligraphy";
+    homepage = "https://github.com/cyrealtype/lora";
+    license = licenses.ofl;
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:36]:
```suggestion
    license = lib.licenses.ofl;
```    
- #208242
- #293767

File: pkgs/by-name/lo/lora/package.nix [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    platforms = platforms.all;
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:37]:
```suggestion
    platforms = lib.platforms.all;
```   
- #208242
- #293767

File: pkgs/by-name/lo/lora/package.nix [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    maintainers = with maintainers; [ ofalvai ];
```
Comment (nyabinary) [inline pkgs/by-name/lo/lora/package.nix:38]:
```suggestion
    maintainers = with lib.maintainers; [ ofalvai ];
```
- #208242
- #293767

**Pull request merged**